slave-base-ubuntu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:12.04.5
+FROM ubuntu:16.04
```

Comment:
@robshelly Unfortunately this one will need to remain 12.04, but there's nothing stopping you creating a separate 16.04 version if you need it. That said, why are you using the ubuntu slave? It's only really here for legacy reasons (3.x builds) and anything else should ideally use a CentOS base image.